lib/ecto_adapters_dynamodb/migration.ex

```diff
@@ -391,6 +392,26 @@ defmodule Ecto.Adapters.DynamoDB.Migration do
     end
   end
 
+  defp set_ttl(_, nil), do: :ok
+  defp set_ttl(table_name, table_options) do
+    if Keyword.has_key?(table_options, :ttl_attribute) do
```

Comment:
Is this conditional necessary? Since `do_set_ttl` is set up to handle a `nil` value in the second argument, it seems to me like it would be ok to simply call `do_set_ttl` without first checking if the key is present, since it'll just be `nil` if it isn't.

Reply:
Strictly, no, I could go without it. However I wanted a clear distinction between the three cases:
* No change to TTL
* Set the TTL
* Remove the TTL

but I wanted to do it without the mess of an `enable` flag in the migration. I figured it was preferable and clearer to allow the user to set `ttl_attribute` to `nil` to remove/disable the TTL, and simply not have the `ttl_attribute` flag at all to cause no change. The trick is that `nil` is also what's returned by `table_options[:ttl_attribute]` or `Map.get(table_options, :ttl_attribute)` when there's no value present, and so I'd have to call `Map.get(table_options, :ttl_attribute, :no_change)` or something, then flip that `:no_change` to `nil` and `nil` to something else and it all got much more complex than what I settled on there.